previews/azure-functions/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Azure Functions
+summary: Azure Functions Preview
+related:
+ - samples/previews/azure-functions
+reviewed: 2020-06-09
+---
+
+Supported triggers
+- [Azure Service Bus](/previews/azure-functions/azure-service-bus.md)
+- [Azure Storage Queues](/previews/azure-functions/azure-storage-queues.md)
+
+
+## Known constraints and limitations
```

Comment:
if we repeat this over and over again, we should make this an include instead.

Reply:
The idea is to split those to specific documents, per package/transport. Limitations for one are different from the limitations for another.